storybookjs/storybook
# Clarify warning message

Fixes #10213 (clarifies docs which are logged to screen for CRA users).

Issue:

## What I did

Clarified language so people less familiar with Storybook don't make the same mistake I did (see #10213) 🤣

## How to test

This is just update to the documentation so presumably it needs no testing. 

Otherwise, to see the effect, CRA users should just ensure that the they have an empty 'addons' field in `.storybook/main.js` as that should trigger the message to be logged.
